6-downloading-and-running-existing-tasks/README.md

```diff
@@ -14,148 +14,147 @@ In this exercise you will explore some existing tasks, including several tasks t
-
-```
-$ bolt task run package action=install name=vim --nodes all --run-as root
-Started on node1...
-Finished on node1:
-  {
-    "status": "present",
-    "version": "2:7.4.160-2.el7"
-  }
-Ran on 1 node in 15.26 seconds
-```
+- Run the 'bolt task show' command to view a list of the tasks installed in the previous exersize.
+
+    ```
+    $ bolt task show
+    apache                        Allows you to perform apache service functions
+    apt                           Allows you to perform apt functions
+    bootstrap                     Bootstrap a node with puppet-agent
+    exec                          Executes an arbitrary shell command on the target system
+    facter_task                   Inspect the value of system facts
+    install_puppet                Install the puppet 5 agent package
+    minifact
+    mysql::sql                    Allows you to execute arbitrary SQL
+    package                       Manage and inspect the state of packages
+    puppet_conf                   Inspect puppet agent configuration settings
+    puppet_device                 Run puppet device on a (proxy) Puppet agent
+    puppeteer::apply              Run a puppet apply on agents
+    puppeteer::certificate_info   Grab certificate information on agent
+    puppeteer::external_fact      Add or update external facts on agents
+    puppeteer::features           Query the node for what features are available
+    puppeteer::providers          Query the node for what providers are available
+    resource                      Inspect the value of resources
+    service                       Manage and inspect the state of services
+    
+    ```
+
+        
+# View and use parameters for a specific task
+
+1. Run `bolt task show package` to view the parameters that the package task uses. 
+
+    ```
+    $ bolt task show package
+    package - Manage and inspect the state of packages
+    
+    USAGE:
+    bolt task run --nodes, -n <node-name> package action=<value> name=<value> [version=<value>] [provider=<value>]
+    
+    PARAMETERS:
+    - action: Enum['install', 'status', 'uninstall', 'upgrade']
+        The operation (install, status, uninstall and upgrade) to perform on the package
+    - name: String[1, default]
+        The name of the package to be manipulated
+    - version: Optional[String[1, default]]
+        Version numbers must match the full version to install, including release if the provider uses a release moniker.
+        Ranges or semver patterns are not accepted except for the gem package provider.
+        For example, to install the bash package from the rpm bash-4.1.2-29.el6.x86_64.rpm, use the string '4.1.2-29.el6'.
+    - provider: Optional[String[1, default]]
+        The provider to use to manage or inspect the package, defaults to the system package manager
+    ```
+2.  Using parameters for the package task, run the bash package and check on the status:
```

Comment:
"run the bash package" isn't quite accurate, since "bash" is the *target* of the package task. Maybe "Using parameters for the package task, check on the status of the bash package"?

Reply:
Fixed